packages/cojson/src/coValues/group.ts

```diff
@@ -123,6 +123,13 @@ function healMissingKeyForEveryone(group: RawGroup) {
 }
 
 function needsKeyRotation(group: RawGroup) {
+  const me = group.core.node.getCurrentAccountOrAgentID();
+
+  // Checking only direct membership because inside the migrations we can't navigate the parent groups
```

Comment:
I don't get why we have this limitation. All parent groups are completely loaded by this point

Reply:
You are right, switched to use `myRole`

Had some issues before related to the way it tries to acces the current account ID, but I fixed that so it is ok now.